Lib/IO_layer_Lib/IO_func.c

```diff
@@ -101,14 +101,23 @@ void _swap_int16_t (uint16_t *a, uint16_t *b)
 	*a = temp;
 }
 
+/**
+  * @brief  fills the screen with a color.
+  * @param  color selects color for background
+  */
```

Comment:
Hierbij nog een retval van de errorcode in Doxygen comment toevoegen neem ik aan?

Reply:
comment bij gevoegd